cmd/ztest/ztest.c

```diff
@@ -85,6 +85,7 @@
  * ztest can invoke them during backwards compatibility testing (-B).
  */
 
+#undef NDEBUG
```

Comment:
I think it would be better to do this in the Makefile.am `-U NDEBUG`.

Reply:
Ok will do. I wasn't actually aware of -U for undefing.